test/sampleRecord_test.cpp

```diff
@@ -441,3 +441,28 @@ void SampleRecordTest::testLoggerMessageAlwaysHasTime() {
 
         CPPUNIT_ASSERT_EQUAL(true, tick < 1000);
 }
+
+void SampleRecordTest::test_get_sample_value_by_name()
+{
+		lc->ADCConfigs[7].scalingMode = SCALING_MODE_RAW;
```

Comment:
What is up with your indentation?  Something is not right with your editor.

Reply:
I updated my IDE, and I think it went back to some defaults.  I'll sweep through the entire code with a formatter after this PR